tests/unit/test_database.py

```diff
@@ -0,0 +1,109 @@
+import os
+from typing import List
+
+import boto3
+import pytest
+from moto import mock_dynamodb2
+
+from src.element.database import DynamoDB, Record
+
+sample_record_list: List[Record] = [
```

Comment:
This is fine for now, but ideally we should think about creating a method that will make the creation of this object easier in the future. Something like create_dynamo_record_for_testing( ... ) If we plan on adding more tests to this file in the future adding the method mentioned above will make things easier to follow and read

Reply:
Yes I agree with your suggestion. By any chance, do you know where/how we should put all the sample data?